lms/djangoapps/learner_home/test_serializers.py

```diff
@@ -864,18 +927,49 @@ def test_enrollments(self):
             "suggestedCourses": [],
         }
 
-        input_context = {
-            "resume_course_urls": resume_course_urls,
-            "ecommerce_payment_page": random_url(),
-            "course_mode_info": course_mode_info,
-        }
-
         output_data = LearnerDashboardSerializer(input_data, context=input_context).data
 
         # Right now just make sure nothing broke
         courses = output_data.pop("courses")
         assert courses is not None
 
+    def test_entitlements(self):
+        # One standard enrollment, one fulfilled entitlement, one unfulfilled enrollment
+        enrollments = [
+            self.create_test_enrollment(),
+            self.create_test_enrollment()
+        ]
+        unfulfilled_entitlements = [CourseEntitlementFactory.create()]
+
+        input_context = self.make_test_context(
+            enrollments=enrollments,
+            enrollments_with_entitlements=[enrollments[1]],
+            unfulfilled_entitlements=unfulfilled_entitlements,
+        )
+
+        input_data = {
+            "emailConfirmation": None,
+            "enterpriseDashboards": None,
+            "platformSettings": None,
+            "enrollments": enrollments,
+            "unfulfilledEntitlements": unfulfilled_entitlements,
+            "suggestedCourses": [],
+        }
+
+        output_data = LearnerDashboardSerializer(input_data, context=input_context).data
+
+        courses = output_data.pop("courses")
+        #We should have three dicts with the same base structure
```

Comment:
Nit: potentially worth some additional variables or comments to make it clear what all these asserts are doing